### DISCUSSION TITLE
Meal Plan Rules with multiple categories (but not all at the same time)

### DISCUSSION THREAD

**Discussion by tmllull** [mealie-recipes/mealie#1680] [General]
Hi, I'm trying the Meal Plan Rules, and I don't know if I'm using wrong, or this options is not implemented. I try to add some rules, for example I want to eat a kind of recipe from "Category1" OR "Category2".

I add 1 rule, adding both categories (1 and 2), but always say that no recipes found. I tried to add 2 separated rules (one for each category), with the same result. But if I put only one rule with one category, works fine. So, I suppose that the criteria is an AND, not OR.

Is there any way to do this?

**Comment by hay-kot** [comment]
Not currently. I pictured it being used sort of like this. Users would have overarching categories like Dinner, Lunch, Sides, and Breakfast. Those would be the general rules and then you could apply more specific rules on a given day, like on Sundays I like to eat fish, so I'd use the Fish tag.

**Reply by tmllull** [reply]
Oh, I see. I use categories for "Meat", "Fish", "Pasta"... And tags for "Pork", "Tuna", "Eggs"... So, with tags it's not possible neither, wright?

Well, I take a review on how I use the tags/categories.

Thanks for your help!
